### DISCUSSION TITLE
Google analytics tag for single html files

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#702] [Q&A] [answered]
For web sites I see how to embed google analytics tags here: https://quarto.org/docs/websites/website-tools.html#google-analytics

How would one do this for a standalone web page when I'm not using Quarto's web site capabilities?

**Comment by dragonstyle** [comment, marked as answer]
We currently only support this for websites, though you could certainly accomplish something similar by placing the Google Analytics script for your page in the header of the page using `include-in-header` (a path to a file) or `header-includes` (inline in the document front matter)

https://quarto.org/docs/output-formats/html-basics.html#includes

**Comment by espinielli** [comment]
Is there a way to conditionally include?
I have a HTML book which I would like to use google analytics only in production...and the suggestion from @dragonstyle would be fine.
BUT we use Netlify also for previews out of pull requests which should not include the GA bits...

I was thinking to exclude the inclusion if the branch name is not main/master but clearly I cannot have inline R in the _quarto.yml ;-)

Anybody has/had a similar need or any suggestions?
